src/main/kotlin/org/rust/ide/inspections/RsUnresolvedReferenceInspection.kt

```diff
@@ -66,39 +65,60 @@ class RsUnresolvedReferenceInspection : RsLocalInspectionTool() {
                 val context = AutoImportFix.findApplicableContext(holder.project, methodCall)
 
                 if (!isMethodResolved || context != null) {
-                    holder.registerProblem(methodCall, context)
+                    holder.registerProblem(methodCall, listOfNotNull(
+                        getAutoImportFix(methodCall, context)
+                    ))
                 }
             }
         }
 
     private fun RsProblemsHolder.registerProblem(
         element: RsReferenceElement,
-        context: AutoImportFix.Context?
+        fixes: List<LocalQuickFix>
     ) {
-        val candidates = context?.candidates
-        if (candidates.isNullOrEmpty() && ignoreWithoutQuickFix) return
+        if (fixes.isEmpty() && ignoreWithoutQuickFix) return
```

Comment:
`ignoreWithoutQuickFix` was invented to reduce the number of false-positives. We guess that if there is an item that can be auto-imported, then it's not a false-positive. But, these heuristics lead to false-positives anyway... We don't want to make more. 

False-positives caused poor name resolution & macro expansion implementations (we're planning a new name resolution design now).

So, I suggest not to change the current heuristics behavior but make an intention instead of quickfix! An intention applied to an unresolved reference.

Reply:
So the unresolved function call should not be annotated as an error, and there should instead be an intention that will be offered on unresolved function calls? And the code of the intention that checks if the function call is unresolved will be shared with this inspection?

Reply:
Yep. But I'm not sure there are a lot of code to share. `path.reference?.multiResolve()?.isEmpty() != false`

Reply:
Ok, I changed it to an intention. Now that it's an intention, I guess that we can also support method calls?

Reply:
Yes!